test/testenv/verificationutils.go

```diff
@@ -489,3 +500,45 @@ func VerifySplunkServerConfSecrets(deployment *Deployment, testenvInstance *Test
 		}
 	}
 }
+
+// VerifyPVCExists verifies if PVC exist
```

Comment:
`VerifyPVCExists` vs `VerifyPVC` as we testing existing  or not existing depending on value of `expectedtoExist`